qa/deepsea/deepsea.yaml

```diff
@@ -12,3 +12,6 @@ tasks:
         - deepsea.ceph_conf:
         - deepsea.orch:
                 stage: 3
+        - exec: 
+                mon.a:
+                - 'ceph config set mon auth_allow_insecure_global_id_reclaim false || true'
```

Comment:
Do we really need to have `|| true` at the end?

Reply:
Yes?

Reply:
Are you asking the reason why it is there? It's because the test might run on a version of ceph older than 14.2.20, in which case we don't want to fail the test just becase this feature is missing.